strongbox-client/pom.xml

```diff
@@ -121,10 +121,16 @@
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
         </dependency>
-    
+
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
```

Comment:
@ptirador I think scope can be extracted into parent `pom.xml`

Reply:
:+1: